packages/chakra-ui-docs/pages/divider.mdx

```diff
@@ -0,0 +1,40 @@
+# Divider
+
+Dividers are used to display a thin horizontal line.
```

Comment:
We could say "horizontal or "vertical line". I've just pushed a change to support vertical dividers. Thanks